Lib/test/test_cmd_line_script.py

```diff
@@ -648,7 +648,7 @@ def test_syntaxerror_invalid_escape_sequence_multi_line(self):
             self.assertEqual(
                 stderr.splitlines()[-3:],
                 [   b'    foo = """\\q"""',
-                    b'                 ^',
+                    b'          ^',
```

Comment:
Does this work also with the old parser?

Reply:
It does with `python -X oldparser -m test -v test_cmd_line_script`. I also checked on the REPL that the `SyntaxError` emitted has the same `col_offset`.